### ISSUE TITLE
Should depend on JRE11

### ISSUE THREAD

**Issue by Tea23** [Adamcake/bolt-flatpak#2]
As well documented in Bolt itself, HDOS doesn't run in JRE17. The HDOS devs only support JRE11 at present and won't fix the 17 breakage. JRE11 should be included for HDOS support. Bolt/HDOS run okay with 11 installed instead.

**Comment by Adamcake** [comment]
HDOS has a trivial mistake in its getdown config that prevents the program from starting on Java 17. It would take the developers about ten seconds to fix. When fixed, HDOS works perfectly, and I know this for a fact because I launched it manually with this configuration and was able to play the game without any issues (image proof attached.)

The HDOS devs being unable to fix a simple typo is not a good enough reason to downgrade all OSRS players by six major Java versions. Although if they need help reading their own log files, feel free to send them my way.
![image](https://github.com/Adamcake/bolt-flatpak/assets/30269600/fa5bf34c-77d3-45d2-90c1-637e456f852b)
